### DISCUSSION TITLE
feat: Add WeChat as a channel

### DISCUSSION THREAD

**Discussion by pranavrajs** [chatwoot/chatwoot#8189] [Feature Requests]
- [ ] Admins should be able to configure WeChat app for Chatwoot
- [ ] Support messages for WeChat

**Comment by robinWongM** [comment]
Hi, I'd love to work on it but I ran into a problem. (Sorry that I'm not quite familiar with RoR) Is there anything like abstract channel interface in ChatWoot? If so, I could extend it as a WeChat channel rather than from scratch.

**Comment by pranavrajs** [comment]
@sojan-official would be able to help. Can you please join our Discord channel for discussions (https://discord.gg/cJXdrwS)?

**Comment by ruidacn** [comment]
> Hi, I'd love to work on it but I ran into a problem. (Sorry that I'm not quite familiar with RoR) Is there anything like abstract channel interface in ChatWoot? If so, I could extend it as a WeChat channel rather than from scratch.

How is the progress? Would like to help if needed.

**Comment by ruidacn** [comment]
pls count me in. Be happy to contribute.

**Comment by ZheZhang10** [comment]
Willing to help

**Comment by sojan-official** [comment]
@ZheZhang10  let us know if you need anything from your end !

**Comment by pranavrajs** [comment]
Moving this to feature discussion as the feature is not planned in the immediate roadmap.
